packages/protocol/contracts/common/UsingFixidity.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.5.8;
+
+import "fixidity/contracts/FixidityLib.sol";
+
+
+contract UsingFixidity {
+  using FixidityLib for int256;
+
+  int256 public constant FIXED1 = 1000000000000000000000000;
+  int256 public constant FIXED_HALF = 500000000000000000000000;
+
+  function toFixed(uint256 n) internal pure returns (int256) {
+    return int256(n).newFixed();
```

Comment:
this looks wrong; if i understand correctly. doing `int256(n)` where n is a `uint256` can interpret as a negative number a certain group of `uint`s.

Because, i think this "cast", but doesn't "converts"

Reply:
Yeah, should add an overflow check here.